src/com/rsyslog/slfa/model/LogFile.java

```diff
@@ -57,48 +56,44 @@ private void init(InputStreamReader reader, ArrayList<AnonType> typelist) {
             list.get(i).onFileStart();
         }
         fileRd = new BufferedReader(reader);
-        rand = new Random(System.currentTimeMillis());
+    }
+
+    public String readLineFromInput() {
```

Comment:
Not sure if I overlook something obvious here, but I think since this function is only used in this class, it'd be better to make it private instead of public.

Reply:
i have another change in queue which reworks LogFile, this will be included.

Reply:
ok

Reply:
did include the small change here as I saw there are anyway new commits on master branch now

Reply:
Thanks. Could you squash the two commits please? Also FYI: Not really sure why static analyzer is taking so long, was much faster on another PR i just made...

Reply:
Not worked with lgtm in any project yet, buts nothing we can control.